cobigen-setup/files/commandsAndOptions.md

```diff
@@ -0,0 +1,19 @@
+* [generate, g]: Command used for code generation.
+To know more about implementation of this command you can follow https://katacoda.com/devonfw/scenarios/cobigen. Here we have shown the implementation "generate" command for further code generation.
+
+* [adapt-templates, a]: Generates a new templates folder next to the cobigen cli and stores its location    inside  a configuration file. After executing this command, the CLI will attempt to use the specified Templates folder.
+
+`cobigen a`{{execute T1}}
+
+{
+   what it will do: Downloads the latest CobiGen_Templates and unpacks them next to the CLI. CobiGen will from now on use these unpacked Templates for generation.
+}
+    - < --custom-location, -cl > : Allows the user to choose an absolute file path to a custom location where the CobiGen Templates should be stored and read from.

```

Comment:
The '<' and '>' is displayed as &lt; and &gt; in Katacoda. Seems that Katacoda can't display them well. You should choose an other notation. Maybe it works if you wrap this part in an `` block